### PR TITLE
[release-8.1] Handle pows received before POW_SUBMISSION state

### PR DESCRIPTION
**Pull request by sandipbhoir**
## Backward Compatibility

- [x] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [ x] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

**Review by chetan-zilliqa — APPROVED** [review]

File: src/libDirectoryService/FinalBlockPostProcessing.cpp [hunk 1]
```diff
@@ -285,18 +292,16 @@ void DirectoryService::ProcessFinalBlockConsensusWhenDone() {
   }
   m_mediator.m_node->ClearUnconfirmedTxn();
 
+  AccountStore::GetInstance().InitTemp();
+  AccountStore::GetInstance().InitRevertibles();
```
Comment (KaustubhShamshery) [inline src/libDirectoryService/FinalBlockPostProcessing.cpp:296]: was this change intended?

Comment (sandipbhoir) [reply src/libDirectoryService/FinalBlockPostProcessing.cpp:296]: Yes. Basically we reverted the previous change. Now, Expiry window flag & state check in ProcessPowSubmission will allow submission to be accepted while node is still FinalBlockConsensus state

**Review by KaustubhShamshery — APPROVED** [review]
